decisions/backend/0016-useful-sql-queries.md

```diff
@@ -23,7 +23,7 @@ WITH aUsers AS (
     FROM review
     WHERE ready
 )
-SELECT u.id, u.nick, count(*) AS followers
+SELECT u.id, u.nick, u.name, count(*) AS followers
```

Comment:
Have you tried to launch this query after modification?

Reply:
khm-khm! of course. Who d0 you think i am?! A rookie?